Jenkinsfile

```diff
@@ -11,5 +11,32 @@ pipeline {
         sh 'xcodebuild -workspace GiniSwitchSDK/Example/GiniSwitchSDK.xcworkspace -scheme "GiniSwitchSDK-Example" -destination \'platform=iOS Simulator,name=iPhone 6\' test'
       }
     }
+    stage('Hockey deployment') {
+      when {
+        branch 'feature-inject-creds'
```

Comment:
I just realized about this. Are you sure you want to deploy to Hockey in this branch? 😛

Reply:
I guess that's for testing purpose.

Reply:
I guess so

Reply:
What a mess. Thanks for noticing!